portfolio/src/main/webapp/script.js

```diff
@@ -15,20 +15,40 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Arms for your chairs are just chairs for your arms.', 'I own chickens', 'sup and wassup', 'yeet'];
+// function addRandomGreeting() {
```

Comment:
Remove the commented-out code. (It's OK to have them when you're developing the solution, but once you're done, remove them. You can always see previous versions of the code using Git, since Git stores all versions.)

Reply:
Did this!